src/main/java/edu/cornell/kfs/sys/businessobject/lookup/CreateDoneBatchFileLookupableHelperServiceImpl.java

```diff
@@ -16,12 +22,36 @@
 import org.kuali.rice.kim.api.identity.Person;
 
 import edu.cornell.kfs.sys.CUKFSPropertyConstants;
+import edu.cornell.kfs.sys.batch.CreateDoneBatchFile;
 import edu.cornell.kfs.sys.batch.service.CreateDoneBatchFileAuthorizationService;
 
 public class CreateDoneBatchFileLookupableHelperServiceImpl extends BatchFileLookupSearchServiceImpl {
+	private static final Logger LOG = LogManager.getLogger(CreateDoneBatchFileLookupableHelperServiceImpl.class);
 
 	protected CreateDoneBatchFileAuthorizationService createDoneAuthorizationService;
 
+	@Override
+	public List<BusinessObjectBase> getSearchResults(Class<? extends BusinessObjectBase> businessObjectClass,
+			MultivaluedMap<String, String> fieldValues) {
+
+		List<BusinessObjectBase> results = super.getSearchResults(businessObjectClass, fieldValues);
+		List createDoneBatchFiles = new ArrayList<CreateDoneBatchFile>();
+		for (Object file : results) {
+			BatchFile batchFile = (BatchFile) file;
+			CreateDoneBatchFile createDoneBatchFile;
+			try {
+				createDoneBatchFile = new CreateDoneBatchFile(batchFile.getId());
+				createDoneBatchFiles.add(createDoneBatchFile);
+			} catch (FileNotFoundException e) {
+				LOG.error(
+						"An error has occured while creating a CreateDoneBatchFile from the BatchFile object in the search results: "
```

Comment:
The method name should be added to the error message.

Reply:
I will add the method name.